.doc_gen/metadata/sqs_metadata.yaml

```diff
@@ -178,9 +179,9 @@ sqs_ListQueues:
         - sdk_version: 1
           github: rust_dev_preview/sqs
           excerpts:
-            - description:
+            - description: Retrieve the first &SQS; queue in the region's list.

```

Comment:
region's list. --> Region list.

Reply:
No - this is possessive of the singular region. 

"Retrieve the first &SQS; queue listed in the region."?